src/lang/en.json

```diff
@@ -3,18 +3,18 @@
   "block": [
     {
       "greeting": "Hi, there",
-      "question": "You may be wondering, who is Mahsa Amini?",
-      "answer": "She was a 22-year-old Iranian who <u>died in police custody</u> after being severely beaten for not wearing a proper hijab."
+      "question": "You may be wondering, who Mahsa Amini is",
```

Comment:
I feel like the Question style of this sentence is better: it creates a question mark in your mind when you read it. And it makes it clear what's going to be explained next; namely, who is Mahsa Amini?